sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_container_registry_client.py

```diff
@@ -186,50 +186,546 @@ async def get_next(next_link=None):
+        except ResourceNotFoundError:
+            pass
+
+    @distributed_trace_async
+    async def delete_tag(self, repository: str, tag: str, **kwargs: Any) -> None:
+        """Delete a tag from a repository. If the tag cannot be found or a response status code of
+        404 is returned an error will not be raised.
+
+        :param str repository: Repository the tag belongs to
+        :param str tag: The tag to be deleted
+        :returns: None
+        :rtype: None
+        :raises: :class:`~azure.core.exceptions.ResourceNotFoundError`
+
+        Example
+
+        .. code-block:: python
+            from azure.containerregistry.aio import ContainerRepositoryClient
+            from azure.identity.aio import DefaultAzureCredential
+            account_url = os.environ["CONTAINERREGISTRY_ENDPOINT"]
+            client = ContainerRepositoryClient(account_url, "my_repository", DefaultAzureCredential())
+            async for artifact in client.list_tags():
+                await client.delete_tag(tag.name)
+        """
+        try:
+            await self._client.container_registry.delete_tag(repository, tag, **kwargs)
+        except ResourceNotFoundError:
+            pass
+
+    @distributed_trace_async
+    async def get_manifest_properties(
+        self, repository: str, tag_or_digest: str, **kwargs: Any
+    ) -> ArtifactManifestProperties:
+        """Get the properties of a registry artifact
+
+        :param str repository: Name of the repository
+        :param str tag_or_digest: The tag or digest of the manifest
+        :returns: :class:`~azure.containerregistry.ArtifactManifestProperties`
+        :raises: :class:`~azure.core.exceptions.ResourceNotFoundError`
+
+        Example
+
+        .. code-block:: python
+            from azure.containerregistry.aio import ContainerRepositoryClient
+            from azure.identity.aio import DefaultAzureCredential
+            account_url = os.environ["CONTAINERREGISTRY_ENDPOINT"]
+            client = ContainerRepositoryClient(account_url, "my_repository", DefaultAzureCredential())
+            async for artifact in client.list_manifests():
+                properties = await client.get_registry_artifact_properties(artifact.digest)
+        """
+        if _is_tag(tag_or_digest):
+            tag_or_digest = await self._get_digest_from_tag(repository, tag_or_digest)
+
+        return ArtifactManifestProperties._from_generated(  # pylint: disable=protected-access
+            await self._client.container_registry.get_manifest_properties(repository, tag_or_digest, **kwargs),
+            repository_name=repository,
+            registry=self._endpoint,
         )
-        return ContainerRepository(
-            self._endpoint, repository_name, credential=self._credential, pipeline=_pipeline, **kwargs
+
+    @distributed_trace_async
+    async def get_tag(self, repository: str, tag: str, **kwargs: Any) -> ArtifactTagProperties:
```

Comment:
Similarly to manifest properties - would it make sense to rename this to get_tag_properties?

Reply:
This rename was one of the archboard feedback comments listed [here](https://github.com/Azure/azure-sdk/issues/2832#issuecomment-843639721)